src/ui/hooks/users.ts

```diff
@@ -43,16 +43,22 @@ export type UserInfo = {
 };
 
 export enum Nag {
-  FIRST_REPLAY = "first_replay",
   FIRST_REPLAY_2 = "first_replay_2",
   FIRST_BREAKPOINT_EDIT = "first_breakpoint_edit",
   FIRST_BREAKPOINT_ADD = "first_breakpoint_add",
-  FIRST_BREAKPOINT_REMOVED = "first_breakpoint_removed",
+  FIRST_BREAKPOINT_SAVE = "first_breakpoint_save",
   FIRST_CONSOLE_NAVIGATE = "first_console_navigate",
-  FIRST_GUTTER_CLICK = "first_gutter_click",
   DOWNLOAD_REPLAY = "download_replay",
 }
 
+// Keeping a list of unused nag types here so we don't accidentally
```

Comment:
Since the way we use nags is accommodating, we should keep a record of previously used nags so we don't accidentally reuse them. Probably should move it to its own file and be more liberal with comments...

Reply:
makes sense